webapp/channels/src/components/team_controller/team_controller.tsx

```diff
@@ -53,6 +54,7 @@ function TeamController(props: Props) {
     useTelemetryIdentitySync();
 
     useEffect(() => {
+        InitialLoadingScreen.stop();
```

Comment:
In Web Guild meeting, we discussed how to ensure that the loading screen doesn't get stuck open on a screen that doesn't call this. The solution we landed on that seemed like our best bet was to make it so that the `root.tsx` takes care of calling this method except when Channels is mounted so we can have something in Channels take care of it at a better time